tokio-trace-macros/example/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "example"
+version = "0.1.0"
+authors = ["David Barsky <dbarsky@amazon.com>"]
+
+[dependencies]
```

Comment:
Note that if you want the examples to live inside crate rather than in their own crate, you can just put these deps in the crate's `[dev-dependencies]` section.

Reply:
Hmm, good point—I'll explore this.

Reply:
Fixed.